lib/orocos/process.rb

```diff
@@ -1171,9 +1198,25 @@ def self.wait_running(process, timeout = nil, name_service = Orocos::CORBA.name_
                     block.call(process)
                 else
                     # Get any task name from that specific deployment, and check we
-                    # can access it. If there is none
+                    # can access it either through the ior pipe or through the CORBA name
+                    # service.
+                    if ior_read?
```

Comment:
process.alive? implies having the pipe

I don't completely understand the rationale behind keeping the name service method.

Reply:
Didnt we agree on keeping the old behavior? I might have misunderstood that, if we are removing the name service at all, I'll have to change where the pipe is created because of this https://github.com/rock-core/tools-orocosrb/blob/83c4369cfc0d9097ff2cc492c59a0a0c674b3569/lib/orocos/process.rb#L826 as I cant get the pipe from that scope.